pom.xml

```diff
@@ -25,6 +25,7 @@
         <project.root>${basedir}</project.root>
         <java.version>12</java.version>
         <checkstyle.config>${project.root}/src/main/resources/checkstyle.xml</checkstyle.config>
+        <springboot.version>2.1.6.RELEASE</springboot.version>
```

Comment:
The parent declaration is still specifying the same version explicitly, so we have the version number duplicated in two places. Let's see if we could change the parent declaration to use the variable instead:

```
<parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
        <version>2.1.6.RELEASE</version>
        <relativePath/> <!-- lookup parent from repository -->
    </parent>```

Reply:
I had tried to use that, unfortunately maven doesn't allow for properties to be used in parent definitions.